### DISCUSSION TITLE
How to override EditLink button ?

### DISCUSSION THREAD

**Discussion by TechAspirant** [serenity-is/Serenity#5498] [Help]
In my case I have multiple edit link in grid and I wanted to override it from typescript ?

**Comment by VictorTomaili** [comment]
You can hide editlink and add your custom one.

**Reply by TechAspirant** [reply]
Any example ?

**Reply by arunkumar12712** [reply]
https://serenity.is/demo/BasicSamples/CustomLinksInGrid

**Reply by TechAspirant** [reply]
I want to open the same edit dialog which gets open on edit link and pass data from grid to dialog.
